extensions/query-store/.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
+	"parserOptions": {
+		"project": "./extensions/query-store/tsconfig.json"
+	},
+	"rules": {
+		"@typescript-eslint/no-floating-promises": [
+			"error",
+			{
+				"ignoreVoid": true
+			}
+		],
+		"@typescript-eslint/explicit-function-return-type": ["warn"]
```

Comment:
Why warn? We should be enabling everything we can when starting fresh - and unless you're explicitly running eslint yourself (which most people don't do) then you might as well just disable it since no one will notice it.

Reply:
I thought might as well turn it on while there's no code, but didn't realize it doesn't show locally and was only showing for me since I had an eslint extension installed in vscode. Turned it off now.